terraform/hydrocron-dynamo.tf

```diff
@@ -11,7 +11,17 @@ resource "aws_dynamodb_table" "hydrocron-swot-reach-table" {
     name = "range_start_time"
     type = "S"
   }
-
+  attribute {
+    name = "granuleUR"
```

Comment:
Can you provide some sample queries that show how to query the granuleUR?

Reply:
@torimcd just confirming, did you add the info to the wiki?

Reply:
yes! there's now a section on how to query on granuleUR via the AWS CLI